### ISSUE TITLE
Autocomplete value invalid value

### ISSUE THREAD

**Issue by kevinpoot** [a11y-collective/demos#8]
https://github.com/a11y-collective/demos/blob/bcffa04888f61ae0f2c1cac31116b7463fb86dab/en/aria/role-name-value.html#L143

Should be either 'on' or 'off'

**Comment by kevinpoot** [comment]
https://github.com/a11y-collective/demos/blob/bcffa04888f61ae0f2c1cac31116b7463fb86dab/en/aria/accordion.html#L26

**Comment by kevinpoot** [comment]
https://github.com/a11y-collective/demos/blob/bcffa04888f61ae0f2c1cac31116b7463fb86dab/en/aria/forms.html#L32

**Comment by rianrietveld** [comment]
@kevinpoot 
THis is a new guideline in WCAG 2.1 AA
https://www.w3.org/WAI/WCAG21/quickref/#identify-input-purpose
HTML specs
https://www.w3.org/TR/html52/sec-forms.html#sec-autofill
https://www.w3.org/TR/WCAG21/#input-purposes
